ios/Modules/ShareMenuReactView.swift

```diff
@@ -131,11 +135,17 @@ public class ShareMenuReactView: NSObject {
 
                 callback(text, "text/plain", nil)
             }
+        }  else if (dataProvider != nil) {
+            dataProvider.loadItem(forTypeIdentifier: kUTTypeData as String, options: nil) { (item, error) in
+                let url: URL! = item as? String
```

Comment:
```suggestion
                let url: URL! = item as? URL
```

Reply:
@himmelarthur, can you fix this cast so that this PR can be merged? Thanks for this btw